code/obj/flock/structure/flock_structure_parent.dm

```diff
@@ -175,6 +177,8 @@
 
 	takeDamage(damtype, W.force)
 
+	user.lastattacked = src
```

Comment:
Why the whitespace?

Reply:
No reason in particular, I'll remove it if that's better